400-claims.md

```diff
@@ -78,9 +84,9 @@ Source: [400.01-claim.json](examples/400.01-claim.json)
 
 - bundle: The bundle, as defined in [the Bundle Definition](101-bundle-json.md).
 - bundleReference (OPTIONAL): A canonical reference to the bundle used in the last action. This bundle reference SHOULD be digested to identify a specific version of the referenced bundle.
-- created: A timestamp indicating when this release claim was first created. This MUST not be changed after initial creation.
+- created: A timestamp indicating when this claim was first created. This MUST not be changed after initial creation.
 - custom: A section for custom extension data applicable to a given runtime.
-- modified: A timestamp indicating the last time this release claim was modified.
+- modified: A timestamp indicating the last time this claim was modified.
```

Comment:
Since we say that the created time MUST not change, maybe we should use the same language here:

```suggestion
- modified: A timestamp indicating the last time this claim was modified. This MUST be updated each time the claim is modified.
```